### PR TITLE
Generate internalized planning IR

### PR DESCRIPTION
**Pull request by RCHowell**
## Relevant Issues
N/A

## Description

Generates internalized planning IR. This simply replaces all public modifiers in the generated source with internal visibility modifiers. The only thing is that you cannot mark interface methods as internal (only the class) hence the skipping of override/fun visit.

## Other Information
- Updated Unreleased Section in CHANGELOG: **[YES/NO]**
No

- Any backward-incompatible changes? **[YES/NO]**
No

- Any new external dependencies? **[YES/NO]**
  - < If YES, which ones and why? >
  - < In addition, please also mention any other alternatives you've considered and the reason they've been discarded >
 No

- Do your changes comply with the [Contributing Guidelines](https://github.com/partiql/partiql-lang-kotlin/blob/main/CONTRIBUTING.md)
  and [Code Style Guidelines](https://github.com/partiql/partiql-lang-kotlin/blob/main/CODE_STYLE.md)? **[YES/NO]**
Yes

## License Information

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: partiql-planner/src/main/resources/builtins.sql [hunk 1]
```diff
@@ -1,1681 +0,0 @@
--- [trim] ---------
```
Comment (alancai98) [inline partiql-planner/src/main/resources/builtins.sql:1]: Just curious why are these *.sql files getting deleted?

File: partiql-planner/build.gradle.kts [hunk 2]
```diff
@@ -77,3 +77,54 @@ publish {
     name = "PartiQL Planner"
     description = "PartiQL's Experimental Planner."
 }
+
+// Generate IR
```
Comment (alancai98) [inline partiql-planner/build.gradle.kts:81]: nit: could make more clear this is for the internal IR

```suggestion
// Generate internal IR
```

File: partiql-planner/src/main/resources/builtins.sql (hunk 1 above)
Comment (RCHowell) [reply partiql-planner/src/main/resources/builtins.sql:1]: They're inconsequential, outdated, and shouldn't have been pushed to begin with.

File: partiql-planner/build.gradle.kts (hunk 2 above)
Comment (RCHowell) [reply partiql-planner/build.gradle.kts:81]: ack

**Review by alancai98 — APPROVED** [review]

**Pull request merged**